manifests/mock/puppetlabs_mocks.pp

```diff
@@ -1,5 +1,5 @@
 class rpmbuilder::mock::puppetlabs_mocks (
-  $fedora_releases  = ["14","17","20","21","22"],
+  $fedora_releases  = ["14"],
```

Comment:
Why don't we add new ones?

Reply:
we don't actually use fedora mocks for anything anymore (except maybe some legacy cumulus stuff), so that would just be more work to keep up to date later.